tests/multiparser_vue/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -35,24 +33,23 @@ exports[`template-class.vue 1`] = `
   </h2>
 </template>
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-
 <template>
 
+  <h2
+    class="title"
+    :class="{ 'issue-realtime-pre-pulse': preAnimation,
+ 'issue-realtime-trigger-pulse': pulseAnimation}"
+    v-html="titleHtml"
+  >
```

Comment:
I wouldn't say this is the preferred way to write Vue interpolations.

Reply:
We started working on a full html pretty printer and added this vue file as an example. But it’s far from being usable so instead this PR just prints the html part as is and only reformats the js and css pieces.